solutions/usaco-916.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: usaco-916
+source: USACO Bronze 2019 February
+title: The Great Revegetation
+author: Ryan Chou
+---
+## Solution 1
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_revegetate_bronze_feb19.html)
+
+## Solution 2
+We can iterate through the favorite pastures, and increment one to the pasture with the greater index, making the result minimal.
```

Comment:
increment what by one? probably mention that you start with all 1s for the grass types, then increment, etc.